Assets/_BForBoss/_Entities/Scripts/DummyTargetBehaviour.cs

```diff
@@ -9,22 +8,29 @@ public class DummyTargetBehaviour : LifeCycleBehaviour
         private const float DEATH_ANIMATION_DURATION = 5f;
         private const string IS_DEAD = "IsDead";
         private const string HIT = "Hit";
+        private const float MAX_DISSOLVE = 1f;
         private readonly int DEATH_ID = Animator.StringToHash(IS_DEAD);
         private readonly int HIT_ID = Animator.StringToHash(HIT);
+        private readonly int DISSOLVE_ID = Shader.PropertyToID("_Dissolve");
+        
         
-        private Animator _animator;
         
+        private Animator _animator;
+        private Renderer _renderer;
         
         protected override void Awake()
         {
             base.Awake();
             _animator = GetComponentInChildren<Animator>();
+            _renderer = GetComponentInChildren<Renderer>();
         }
 
         protected override void LifeCycleFinished()
         {
             _animator.SetBool(DEATH_ID, true);
-            Destroy(gameObject, DEATH_ANIMATION_DURATION);
+            var tween = _renderer.material.DOFloat(MAX_DISSOLVE, DISSOLVE_ID, DEATH_ANIMATION_DURATION * 2);
```

Comment:
Why is there a * 2 here?

Reply:
Setting it to the same time as the animation, I found it faded out a little too fast and you couldn't really see the whole animation

Reply:
I could expose that, probably a better idea than hardcoding it

Reply:
you can name it `DISSOLVE_ANIMATION_DURATION`,  and just set it to the time of your liking.  We can think about the Tween duration separate from the death anim duration.

Reply:
Like it's good to time it with the death, but the value you're using will still belong to the Dissolve tween